src/Widgets/PanelLabel.vala

```diff
@@ -57,7 +70,7 @@ public class DateTime.Widgets.PanelLabel : Gtk.Grid {
 
     private void update_labels () {
         /// TRANSLATORS: Date format in the panel following http://valadoc.org/#!api=glib-2.0/GLib.DateTime.format */
-        date_label.set_label (Services.TimeManager.get_default ().format (_("%a, %b %e")));
+        date_label.set_label (Services.TimeManager.get_default ().format (_(dateFormat)));
```

Comment:
I think dateFormat shouldn't be translatable now

Reply:
Why wouldn't it be? It's still an strftime string, unless the user enters something else, which would be in their own language.

Of course we could check it's a valid string, but why do that? Any part of the string that's not strftime will be output as entered. Maybe that's what the user wants?

Reply:
You can't translate variable, only constant.

Reply:
I'm not sure what you mean. I just tested it with a different language and region and it worked fine.

Reply:
It'll compile and works good. But dynamic variable can not be translated. So calling _() is not necessary.